python-scripting/query-monitoring/query-monitoring.dig

```diff
@@ -0,0 +1,12 @@
+schedule:
+  minutes_interval>: 5
+
++query-monitoring:
+    docker:
+        image: "digdag/digdag-python:3.7.3-stretch"
+    py>: query-monitoring.monitoring
+    _env:
+        td_apikey: ${secret:td.apikey}
+        td_endpoint: https://api.treasuredata.com
```

Comment:
It'd be nice if we would set endpoint as secret as well for other sites customers for Tokyo or EU.

Reply:
updated but all four variables are now required to be specified through `td wf secrets`.